### PR TITLE
BLM program added

### PR DESCRIPTION
**Pull request by MattReimer**
@philipbaileynar please review this carefully. Kind of hard to change when it's done.

### CODE REVIEW

File: Programs/blm/Program.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Program xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../Program.xsd">
+
+  <Name>Bureau of Land Management</Name>
+  <MachineName>BLM</MachineName>
+
+  <MetaData>
+    <Meta name="projectFile">project.rs.xml</Meta>
+    <Meta name="access">PUBLIC</Meta>
```
Comment (MattReimer) [inline Programs/blm/Program.xml:9]: Cool to be public?

File: Programs/blm/Program.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Program xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../Program.xsd">
+
+  <Name>Bureau of Land Management</Name>
+  <MachineName>BLM</MachineName>
```
Comment (MattReimer) [inline Programs/blm/Program.xml:5]: name and machine name oK?

File: Programs/blm/Program.xml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    <Meta name="projectFile">project.rs.xml</Meta>
+    <Meta name="access">PUBLIC</Meta>
+    <Meta name="projectXSDs">https://xml.riverscapes.xyz/Projects/XSD/V1</Meta>
+  </MetaData>
+
+
+  <Definitions>
+    <!--Here's where we say what's allowed for each collection. Can be a list or a regex-->
+    <Collections>
+      <Collection id="COL_HUC8" name="HUC8">
```
Comment (MattReimer) [inline Programs/blm/Program.xml:17]: We're still on the HUC8 system right?

File: Programs/blm/Program.xml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    </Collections>
+
+    <!--Groups are a lot simpler than collections. They can only be a single folder-->
+    <Groups>
+      <Group id="GRP_HUC8" name="HUC8" folder="HUC8" />
+    </Groups>
+
+    <!--Products for now just need ref, name and folder (url is optional and leads to a wiki)-->
+    <Products>
+      <Product id="RSContext" name="Riverscapes Context (HUC8)" folder="RSContext" url="" />
```
Comment (MattReimer) [inline Programs/blm/Program.xml:29]: The net_att program was RSContext only. Do we need more products allowed?

**Review by philipbaileynar — CHANGES_REQUESTED** [review]
Joe requested that the program be called "BLM Montana & Dakotas"

File: Programs/blm/Program.xml (hunk 2 above)
Comment (MattReimer) [reply Programs/blm/Program.xml:5]: BLM is good

File: Programs/blm/Program.xml (hunk 1 above)
Comment (MattReimer) [reply Programs/blm/Program.xml:9]: PRIVATE

File: Programs/blm/Program.xml (hunk 3 above)
Comment (MattReimer) [reply Programs/blm/Program.xml:17]: yes

File: Programs/blm/Program.xml (hunk 4 above)
Comment (MattReimer) [reply Programs/blm/Program.xml:29]: VBET, pyBRAT, BRAT, RVD, confinement

**Comment by joewheaton** [comment]
Can I get access @philipbaileynar please?

**Comment by MattReimer** [comment]
@joewheaton you have access now

**Pull request merged**